enforcer/pom.xml

```diff
@@ -140,6 +140,8 @@
                         <excludes>
                             <exclude>org.apache.logging:*</exclude>
                             <exclude>log4j:*</exclude>
+                            <exclude>org.slf4j:slf4j-log4j12</exclude>
+                            <exclude>org.apache.logging.log4j:log4j-slf4j-impl</exclude>
```

Comment:
Doesn't this captured in the `<exclude>org.apache.logging:*</exclude>` statement

Reply:
No
The reason is that the group ID s are different